aikau/src/main/resources/alfresco/core/topics.js

```diff
@@ -808,6 +829,43 @@ define([],function() {
       SMART_DOWNLOAD: "ALF_SMART_DOWNLOAD",
 
       /**
+       * This can be called to close the StickyPanel.
+       *
+       * @instance
+       * @type {string}
+       * @default
+       * @since 1.0.48
+       *
+       * @event
+       */
+      STICKY_PANEL_CLOSE: "STICKY_PANEL_CLOSE",
```

Comment:
Can we prefix all of the string values with "ALF" please - less likely for any 3rd party string clashes that way